lib/brakes/brakes.cpp

```diff
@@ -0,0 +1,17 @@
+#include "brakes.hpp"
+
+brakes::BrakeClass(const std::uint8_t pin_, const hyped::io::HardwareGpio gpio)
+{
+  pin    = pin_;
```

Comment:
Think you're getting naming conventions mixed up here. Class data members have an underscore at the end, function arguments do not (i.e. should be `pin_=pin`).